src/erb.c

```diff
@@ -226,6 +243,46 @@ static int buildsvi(char *payload, struct erb_svi_head sviHead, struct erb_svi_s
 
     return (LENGTH_SVI_HEAD + nSV * LENGTH_SVI_SV);
 }
+/* build ERB-RTK message -----------------------------------------------------*/
+static void buildrtk(char *payload, struct erb_rtk rtk, const sol_t *sol, const double *rb) {
+    double pos[3];           /* Lat/Lon/Hgt of base station */
+    double baseline_ecef[3]; /* ECEF vector between base position and rover position in meters */
+    double baseline_enu[3];  /* ENU  vector between base position and rover position in meters */
+    uint32_t tow_ms;
+    int week_number;
+    double time_sec;
+    gtime_t time_of_last_baseline;
+    int i;
+
+    if (sol->type == 0) { /* xyz-ecef */
+        ecef2pos(rb, pos);                 /* transform the base ecef position to a geodetic position */
+        for (i = 0; i < 3; i++) 
+            baseline_ecef[i] = sol->rr[i] - rb[i];
+        ecef2enu(pos, baseline_ecef, baseline_enu);
+    } 
+    else {              /* enu-baseline */
+        for (i = 0; i < 3; i++) 
+            baseline_enu[i] = sol->rr[i];
+    }
+    rtk.base_num_sats = sol->ns;    /* Current number of satellites used for RTK calculation */
```

Comment:
I think the `sol->ns` is RTK-related info, so it'll stay here. The STAT message will be fixed to send `numSV` as more general GNSS info and to avoid duplication. Good point.